content/influxdb/v2.0/reference/internals/shards.md

```diff
@@ -0,0 +1,148 @@
+---
+title: InfluxDB shards and shard groups
+description: >
+  Learn the relationships between buckets, shards, and shard groups.
+  InfluxDB organizes time series data in into **shards** when storing data to disk.
+  Shards are grouped into **shard groups**.
+menu:
+  influxdb_2_0_ref:
+    name: Shards & shard groups
+    parent: InfluxDB internals
+weight: 102
+influxdb/v2.0/tags: [storage, internals]
+related:
+  - /influxdb/v2.0/reference/internals/storage-engine/
+  - /influxdb/v2.0/organizations/buckets/
+  - /influxdb/v2.0/reference/cli/influx/bucket/
+---
+
+InfluxDB organizes time series data in into **shards** when storing data to disk.
+Shards are grouped into **shard groups**.
+Learn the relationships between buckets, shards, and shard groups.
+
+- [Shards](#shards)
+- [Shard groups](#shard-groups)
+  - [Shard group duration](#shard-group-duration)
+  - [Shard group diagram](#shard-group-diagram)
+- [Shard life-cycle](#shard-life-cycle)
+  - [Shard precreation](#shard-precreation)
+  - [Shard writes](#shard-writes)
+  - [Shard compaction](#shard-compaction)
+- [Shard deletion](#shard-deletion)
+
+## Shards
+A shard contains encoded and compressed time series data for a given time range
+defined by the [shard group duration](#shard-group-duration).
+All points in a [series](#series) within the shard's time range are stored in the same shard.
+A single shard contains multiple series, one or more [TSM files](#tsm-time-structured-merge-tree) on disk,
+and belongs to a [shard group](#shard-groups).
+
+## Shard groups
+A shard group belongs to an InfluxDB [bucket](/influxdb/v2.0/reference/glossary/#bucket) and contains time series data for a specific time range defined by
+the [shard group duration](#shard-group-duration).
+
+{{% note %}}
+In **InfluxDB OSS**, a shard group typically contains only a single shard.
+In an **InfluxDB Enterprise 1.x cluster**, shard groups contain multiple shards
```

Comment:
```suggestion
In an **InfluxDB Enterprise 1.x cluster**, a shard group contain multiple copies of a shard
```

Reply:
Shards aren't duplicated, they're distributed. So you have multiple different shards in a shard group spread across multiple instances.